Userland/Libraries/LibJS/Bytecode/Op.h

```diff
@@ -176,6 +176,23 @@ class NewString final : public Instruction {
     StringTableIndex m_string;
 };
 
+class NewTypeError final : public Instruction {
```

Comment:
Can we make this a template somehow to avoid having to implement this separately for each error type?